refactor/src/Config.py

```diff
@@ -99,8 +98,8 @@ def __init__(self, hyperparam_file):
         self.EMBmort = data["EMBmort"]["value"]
         self.delay_EMB = data["delay_EMB"]["value"]
         self.delta_EMB = data["delta_EMB"]["value"]
-        self.infection_delay_time_EMB = data["infection_delay_time_EMB"]["value"]
-        self.infection_delay_prob_EMB = data["infection_delay_prob_EMB"]["value"]
+        self.infection_delay_time_EMB = data["infection_delay_time"]["value"]["emb"]["value"]
```

Comment:
this looks hairy. Honestly I'd prefer that infection_delay_time was an object whose key is the treatment (as an enum) and the value is a dict `{"time": float, "prob": float}`.